esphome/components/dsmr/__init__.py

```diff
@@ -50,6 +52,9 @@ async def to_code(config):
     var = cg.new_Pvariable(config[CONF_ID], uart_component)
     if CONF_DECRYPTION_KEY in config:
         cg.add(var.set_decryption_key(config[CONF_DECRYPTION_KEY]))
+    if config[CONF_DEBUG_TELEGRAMS]:
+        cg.add_define("USE_TELEGRAM_DEBUGGER")
```

Comment:
```suggestion
        cg.add_define("USE_DSMR_TELEGRAM_DEBUGGER")
```

Most defines are prefixed by the integration name (I guess that isn't written down anywhere) because macros are global

Reply:
Makes sense, I will update that.
I'll also see what I can add for debugging encrypted telegrams.
I'm helping somebody out from Austria with some issues, and the encryption is one of them. It'd be good to have some more insight in what's going on there.

Reply:
Shouldn't this be `DSMR_USE_TELEGRAM_DEBUGGER` ?

Reply:
Looked over the codebase, and I only see defines with `USE_` at the start. E.g.
* USE_WIFI_WPA2_EAP
* USE_I2C_MULTIPLEXER
* USE_SOCKET_IMPL_LWIP_TCP
* USE_API_NOISE
* etc...
*